src/controllers/user.controller.js

```diff
@@ -0,0 +1,15 @@
+import { User } from '../models/user.js';
+import { emailService } from '../services/email.service.js';
+
+import { v4 as uuidv4 } from 'uuid';
+import { userServices } from '../services/user.service.js';
+
+const getAllActivated = async (req, res) => {
+  const users = await userServices.getAllActivated()
```

Comment:
There is a missing semicolon at the end of this line. While JavaScript can often handle this, it's a good practice to include semicolons to avoid potential issues.